CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 ## Unreleased
 
+* Fix maven badge
+  ([#21](https://github.com/open-telemetry/semantic-conventions-java/pull/21))
+* Fix readme release reference to include -alpha
+  ([#23](https://github.com/open-telemetry/semantic-conventions-java/pull/23))
+* Upgrade to OTEL API 1.31.x
+  ([#26](https://github.com/open-telemetry/semantic-conventions-java/pull/26))
```

Comment:
I generally remove non user-facing changes in -contrib and -instrumentation repos, but I don't mind either way
```suggestion
```